python/composio/tools/toolset.py

```diff
@@ -1384,7 +1384,7 @@ def initiate_connection(
         redirect_url: t.Optional[str] = None,
         connected_account_params: t.Optional[t.Dict] = None,
         *,
-        auth_scheme: AuthSchemeType = "OAUTH2",
+        auth_scheme: t.Optional[AuthSchemeType] = None,
```

Comment:
The docstring for `initiate_connection` should be updated to reflect that `auth_scheme` is now optional and will be fetched from the API if not provided. This helps maintain clear documentation for future developers.